static/css/bureau.less

```diff
@@ -170,5 +171,185 @@
             .cf-icon();
         }
     }
+}
+
+// Used to help center video container
+@share-width__px: 150px;
+@gutter__em: unit(@grid_gutter-width  / @base-font-size-px, em);
+@gutter-half__em: @gutter__em / 2;
+
+.hero__bureau {
+    padding-top: 0;
+
+    .hero_wrapper {
+
+        .respond-to-min(661px, {
```

Comment:
We should probably make a rule like "min is even 0 digit, max is a 9 digit", to make this simpler.